python/core/composer/qgscomposition.sip

```diff
@@ -561,6 +561,10 @@ class QgsComposition : QGraphicsScene
     void addComposerPicture( QgsComposerPicture* picture );
     /** Adds a composer shape to the graphics scene and advices composer to create a widget for it (through signal)*/
     void addComposerShape( QgsComposerShape* shape );
+    /** Adds a composer polygon and advices composer to create a widget for it (through signal)*/
```

Comment:
advices -> advises (also other instances of this typo). I realise it's also present in other lines here, but may as well fix the new stuff :)